src/constants/tooltip.js

```diff
@@ -25,7 +25,7 @@ export const TOOLTIP_FORMAT_TYPES = {
   PERCENTAGE: 'percentage'
 };
 
-export const TOOLTIP_KEY = 'format';
+export const TOOLTIP_KEY = 'id';
```

Comment:
I would like to keep the format as key, so people can pass in custom fomat without an associated id with the default format

Reply:
In some cases we have same formats...

Reply:
I was thinking about adding custom formats in next iteration

Reply:
is it the percent full? I dont like that format anyways. a format should not change the value, it should only change the display of it. I would say we shouldn't have duplicate formats

Reply:
or we allow format to be either a string or a function so you can have `.3f` or you can have
d => d *100

Reply:
Good thoughts

Reply:
So maybe we can use format as ID and if it's not found we will use it's value as D3 format? Still can't use moment.js format this way. Have to add additional field.
Using format value won't let us have it as a function. Since how then we will store it...